server/src/main/java/org/cloudfoundry/identity/uaa/audit/JdbcAuditService.java

```diff
@@ -38,10 +38,10 @@ protected JdbcTemplate getJdbcTemplate() {
     }
 
     @Override
-    public List<AuditEvent> find(String principal, long after) {
```

Comment:
I don't think we need a name and a signature change. That's what is causing this very large diff.

    public List<AuditEvent> find(String principal, long after) {
         return find(principal, after, IdentityZoneHolder.get().getId());
    }

    //new method - do your changes here. 
    public List<AuditEvent> find(String principal, long after, String zoneId) {
    }

We drive this project through TDD. For PRs we're asking a bit more discipline regards that so we can validate the PR itself. 2 commits. The first commit introduces your test (and any interface changes needed). The test should fail when only the first commit is applied. The second commit implements the change, and the test passes when applied.